tests/test-cluster.py

```diff
@@ -259,14 +274,14 @@ def setup_cluster(self):
                 for i in range(0, size):
                     print("Creating machine {}".format(i))
                     vm = VM(backend)
-                    vm.setup(channel_to_test)
```

Comment:
This is the place where you should check for IPv6 support and not inside the VM class.